docs-ref-conceptual/azure-cli-sp-tutorial-3.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Work with Azure service principals using a certificate – Azure CLI | Microsoft Docs
+description:  Use service principals with an existing certificate to gain control over which Azure resources can be accessed.
+manager: jasongroce
+author: daphnemamsft
+ms.author: daphnema
+ms.date: 08/16/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli, seo-azure-cli
+keywords: azure service principal, create service principal azure, create service principal azure cli
+---
+
+# 3 - Create a service principal using certificate-based authentication
+
+When creating a [Service Principal](./azure-cli-sp-tutorial-1.md), you can choose the either password-based or certificate based-authentication. This article details how you can use an existing **certificate** with the service principal to access the Azure Container Registry.
+
+## Why use certificate-based authentication?
+
+We recommend using certificate-based authentication due to the security considerations that password authentication has. Certificate-based authentication enables you to adopt a phishing resistant authentication, by using [conditional access policies](/azure/active-directory/conditional-access/overview), which better protects the user's assets. 
+
+To learn more about why and how certificate based authentication is more secure, see [Azure Active Directory certificate based authentication concepts](/azure/active-directory/authentication/concept-certificate-based-authentication). 
+
+## How do I add certificates to a Service Principal?
+
+We recommend you use Azure CLI to perform certificate-based authentication, however there is also a way to manually assign an existing certificate to a service principal through the Azure portal:
+
+# [Azure CLI](#tab/concepts)
+
+For certificate-based authentication, use the `--cert` parameter. This parameter requires that you hold an existing certificate. Make sure any tool that uses this service principal has access to the certificate's private key. Certificates should be in an ASCII format such as PEM, CER, or DER. Pass the certi897ficate as a string, or use the `@path` format to load the certificate from a file.
+
+> [!NOTE]
+> When using a PEM file, the **CERTIFICATE** must be appended to the **PRIVATE KEY** within the file.
```

Comment:
This statement is wrong. The private key should not be passed: https://github.com/MicrosoftDocs/azure-docs-cli/issues/4734